content/docs/deployment/csminstallationwizard/src/csm-versions/csm-1.6.0.properties

```diff
@@ -1,7 +1,7 @@
 driverVersion=v2.6.0
 vgsnapshotImage=csi-volumegroup-snapshotter:v1.2.0
-replicationImage=dell-csi-replicator:v1.3.0
+replicationImage=dell-csi-replicator:v1.4.0
+podmonImage=podmon:v1.4.0
 authorizationImage=csm-authorization-sidecar:v1.5.0
-migrationImage=dell-csi-migrator:v1.0.0
-powermaxCSIReverseProxyImage=csipowermax-reverseproxy:v2.4.0
```

Comment:
Powermax team has updated the charts with latest versions, please change accordingly.

Reply:
Addressed.